pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
+	if err != nil {
+		return restoreWaitUnknown
+	} else if strings.Contains(string(bytes), "[-w secs]") {
```

Comment:
That seems pretty fragile...

It looks like iptables-restore doesn't even have a --version flag though anyway. And doesn't exit with an error if you pass invalid options. Sigh.

You could check if `iptables-restore -w < /dev/null` prints "invalid option"...